viite-UI/src/view/LinkPropertyForm.js

```diff
@@ -279,7 +280,20 @@
         '<button id="formRoadAddressBrowserButton" class="open-tool-mode-btn btn btn-block btn-primary" style="margin-top: 5px;">Tieosoitteiden katselu</button>' +
         '<button id="formRoadAddressChangesBrowserButton" class="open-tool-mode-btn btn btn-block btn-primary" style="margin-top: 5px;">Tieosoitemuutosten katselu</button>' +
         '</div>';
-      rootElement.append(emptyFormDiv);
+
+      var emptyFormDivForViewMode =
+          '<p class="form form-horizontal"></p>' +
+          '<div class="form-initial-state" id="emptyFormDiv">' +
+          '<button id="formNodesAndJunctionsButton" class="open-tool-mode-btn btn btn-block btn-primary" style="margin-top: 5px;">Solmut ja liittymät</button>' +
+          '<button id="formRoadAddressBrowserButton" class="open-tool-mode-btn btn btn-block btn-primary" style="margin-top: 5px;">Tieosoitteiden katselu</button>' +
+          '<button id="formRoadAddressChangesBrowserButton" class="open-tool-mode-btn btn btn-block btn-primary" style="margin-top: 5px;">Tieosoitemuutosten katselu</button>' +
+          '</div>';
+
+      if (_.includes(startupParameters.roles, 'viite'))
+        rootElement.append(emptyFormDiv);
```

Comment:
For symmetry, 
emptyFormDiv -> 
emptyFormDivForEditMode tai emptyFormDivForEditRole

Tai vaihtoehtoisesti pohdiskelin noiden jakoa kahdeksi diviksi, jossa toisessa katselunappulat ja toisessa editointiroolille näkyvät nappulat, ja sit editointiroolin nappulat annetaan vaan editointiroolilaisille, näkymänappulat kaikille (kahden iffin sisään kuten nyt), tai aina (iffin ulkopuolelle). (Mut sit ollaan jo pelkän nimeämisen ulkopuolella. :) )